ethcore/wasm/Cargo.toml

```diff
@@ -15,13 +15,13 @@ hmac-drbg = "=0.1.2"
 keccak-hash = { path = "../../util/hash" }
 log = "0.3"
 oasis-types = "0.4"
-parity-wasm = "0.31"
-pwasm-utils = "0.2"
+parity-wasm = "0.41"
+pwasm-utils = "0.12"
 sha2 = "0.6"
 vm = { path = "../vm" }
 wasi-types = "0.1"
 wasm-macros = { path = "macros" }
-wasmi = { git = "https://github.com/oasislabs/wasmi" }
+wasmi = { git = "https://github.com/oasislabs/wasmi", branch = "parity-wasm-0.41" }
```

Comment:
we can use master again now, right?

Reply:
I haven't merged it in. It'd be idea to have an end-to-end fix and also test the failing wasm to make sure that everything works before we make changes in 3 repos (that we might have to revert).